Source/Core/Core/HW/GCPadEmu.cpp

```diff
@@ -33,7 +33,7 @@ static const u16 trigger_bitmasks[] = {
 static const u16 dpad_bitmasks[] = {PAD_BUTTON_UP, PAD_BUTTON_DOWN, PAD_BUTTON_LEFT,
                                     PAD_BUTTON_RIGHT};
 
-static const char* const named_buttons[] = {"A", "B", "X", "Y", "Z", _trans("Start")};
+static const char* const named_buttons[] = {"A", "B", "X", "Y", "Z", "Start"};
```

Comment:
Something for a follow-up PR, but this could be changed into a `constexpr std::array` (just like the classic attachments etc. do); along with a range-based for loop down in line 52.